tests/sse-mockserver/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+val kotlin_version: String by project
+
+plugins {
+  application
+  kotlin("jvm")
+}
+
+dependencies {
+  
+  ///////////////////////////
+  // implementation
+
+  // ktor
+  val ktorVersion = "2.0.0"
+
+  implementation(kotlin("stdlib"))
+  implementation("org.jetbrains.kotlinx", "kotlinx-coroutines-jdk8", "1.5.0")
```

Comment:
Not a big deal, but we have a central file for versions in `gradle/dependencies.gradle`, and that you can use with `groovy.util.Eval.x(project, "x.dep.<your dependency>")`